demo/index.html

```diff
@@ -5,7 +5,6 @@
     <meta content='width=device-width, initial-scale=1' name='viewport'>
     <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
     <title>react-calendar-timeline</title>
-    <link rel="stylesheet" type="text/css" href="demo.css">
```

Comment:
While this gives annoying errors in development, it's actually needed in [production](http://namespace.ee/react-calendar-timeline-docs/#/). Not sure how to make both worlds happy...